src/tests/ftest/soak/soak.yaml

```diff
@@ -65,8 +68,7 @@ container_reserved:
 # test_params - Defines the type of test to run and how long it runs
 #               It also defines how many pools and jobs to create
 #               name:                The name of the Avocado testcase
-#               test_timeout:        The overall timeout in secs; soak12 will be
-#                                    changed to 43200 once it is stable
+#               test_timeout:        The overall timeout in hours
```

Comment:
Optional: Is it possible we can be consistent with test_timeout and timeout to both be in same units either secs or hours. Just for convenience sake.

Reply:
I am going to include a change in test.py to accept timeout in hours  something like 48H or 30M or maybe even 2D  for hour, minutes and days